pythonie/meetups/models.py

```diff
@@ -57,4 +58,5 @@ def __str__(self):
     @classmethod
     def future_events(cls):
         today = datetime.now()
-        return cls.objects.filter(time__gt=today).filter(time__lt=next_n_months(today, 3))
+        return cls.objects.filter(
+            time__gt=today).filter(time__lt=next_n_months(today, 3))
```

Comment:
I can't fully remember but I don't believe the Django ORM produces the correct SQL when the two filters are combined. There should be a test case for this however so it might be worth experimenting with. My hunch is that it uses an OR when combined and an AND in it's current form?